chromium_src/components/content_settings/core/common/content_settings.cc

```diff
@@ -22,18 +22,18 @@ constexpr int brave_value(int incr) {
 
 // clang-format off
 #define BRAVE_HISTOGRAM_VALUE_LIST                                        \
-  {ContentSettingsType::BRAVE_ADS, brave_value(0)},                       \
-  {ContentSettingsType::BRAVE_COSMETIC_FILTERING, brave_value(1)},        \
-  {ContentSettingsType::BRAVE_TRACKERS, brave_value(2)},                  \
-  {ContentSettingsType::BRAVE_HTTP_UPGRADABLE_RESOURCES, brave_value(3)}, \
-  {ContentSettingsType::BRAVE_FINGERPRINTING_V2, brave_value(4)},         \
-  {ContentSettingsType::BRAVE_SHIELDS, brave_value(5)},                   \
-  {ContentSettingsType::BRAVE_REFERRERS, brave_value(6)},                 \
-  {ContentSettingsType::BRAVE_COOKIES, brave_value(7)},                   \
-  {ContentSettingsType::BRAVE_SPEEDREADER, brave_value(8)},               \
-  {ContentSettingsType::BRAVE_ETHEREUM, brave_value(9)},                  \
-  {ContentSettingsType::BRAVE_SOLANA, brave_value(10)},                   \
-  {ContentSettingsType::BRAVE_GOOGLE_SIGN_IN, brave_value(11)},           \
+  {ContentSettingsType::BRAVE_GOOGLE_SIGN_IN, brave_value(0)},           \
```

Comment:
@goodov There's a [DCHECK to check if the histogram values are ordered](https://source.chromium.org/chromium/chromium/src/+/main:components/content_settings/core/common/content_settings.cc;l=146?q=khistogramvalue) based on the content settings types enum. With the content_settings_types.h patch in this PR, we move BRAVE_GOOGLE_SIGN_IN enum to before the Brave ones, but then we also need to move it here to be before the histogram values. 

When we add new content settings types that are Desktop + Android, we'll have to change this ordering again :( perhaps there's a simpler way of doing this that I'm missing? In any case, I don't think this will impact functionality - I don't think we actually use this histogram anywhere.

Reply:
tbh I think we should move all our content setting types into the patch. Who knows what else may happen when things go out of sync.

Reply:
@bridiver thoughts?